Created Greek translation

MIP2000 opened the pull request:
Created Greek translation of all files except the potato.js file which i will add later. I followed the rules in the README and i hope that everything is okay. Plaese feel free to suggest any changes that you might want. Thanks a lot!

zekth commented:
Could you please run `yarn lint` ?

MIP2000 commented:
**I have installed node.js 4.4.0 and yarn but when i try to run the command i have an error**:                 `PS C:\Windows\system32> yarn lint
C:\Users\mip20\AppData\Roaming\nvm\v4.4.0\node_modules\yarn\lib\cli.js:46099
  let {
      ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\mip20\AppData\Roaming\nvm\v4.4.0\node_modules\yarn\bin\yarn.js:24:13)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)`


**By the way i finished the translation of potato.js, should i open a new pull request to upload it, or i must wait to resolve this one?**

zekth commented:
Ooh, you can use `nvm use` to set the right node version. It's `11.2`. You can gather all translations in the same PR.

MIP2000 commented:
I added the potato.js, also i updated my node version in 11.2 and now yarn is installed normally. Excuse me but i am new to the whole procedure and i don't understand what to add after i write **yarn lint**. Could you help me a bit about what commands should i run? Thanks again a lot!

zekth commented:
Normally once you do `yarn lint` it formats the code. You just push the modified code.

MIP2000 commented:
Okay so then i just push the URL from my forked repository?

zekth commented:
yes

MIP2000 commented:
I tried but i am getting this error:
**fatal: not a git repository (or any of the parent directories): .git**

Am i missing anything?
I use this command **yarn lint push https://github.com/MIP2000/qmk_configurator.git**

zekth commented:
Accept this PR: https://github.com/MIP2000/qmk_configurator/pull/2 it will fix the issues :)

MIP2000 commented:
Accepted. Thanks a lot for your time.

zekth commented:
@noroadsleft @yanfali rdy for review

GreekGeorge commented:
Η μετάφραση που έκανες φαίνεται πως έγινε στο Google translate.
Αυτή τη στιγμή, πρέπει και θα ξαναγράψω σχεδόν τη κάθε πρόταση και κάθε label, title, help που έβαλες. Χωρίς παρεξήγηση, αλλά μη κάνεις commit για να φανεί ότι έκανες commit σε κάτι. Κάνε commit γιατί μπορείς όντως να βοηθήσεις.

yanfali commented:
@MIP2000 I think the feedback on this PR makes it unlikely we will be moving forward with it. Thank you very much for your contribution, but I will be closing this PR.
